### PR TITLE
KFSPTS-4977: introduce institutional-config.properties and cleanup ov…

### PR DESCRIPTION
**Pull request by bhutchinson**
…erride properties

This is one of two PRs that should be reviewed together, the other one is in the config repo. However, the one in the config repo should be merged first and then the unit test PR job should be re-run on this PR (so that both sets of changes are taken into account) before it is merged.

### CODE REVIEW

File: src/main/resources/institutional-config.properties [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+work.directory=/infra/work
+reports.directory=/infra/work/reports
+staging.directory=/infra/work/staging
+temp.directory=/infra/work/temp
+batch.file.lookup.root.directories=/infra/work/staging;/infra/work/reports
+templates.directory=/infra/work/templates
+attachment.dir.location=/infra/work/attachments
+staging.directory.sys.batchContainer=/infra/work/staging/sys/batchContainer
+data.xml.root.location=/infra/work/staging/workflow
+data.xml.pending.location=/infra/work/staging/workflow/pending
```
Comment (cah292) [inline src/main/resources/institutional-config.properties]: There seems to be several properties that would benefit from "staging.directory" and/or "data.xml.root.location" placeholders. Would it make sense to use such placeholders for some of these directory-related properties, or are these all being defined without placeholders for a specific reason?

Comment (bhutchinson) [inline src/main/resources/institutional-config.properties]: They were all listed out without placeholders in the template file from puppet I was using as reference, and I also wasn't sure about the interaction between the default properties files, the new institition-config.properties and the command-line property overrides file. That said, you're correct that they would probably benefit from placeholders so let me try it out and see what I can do.

File: src/main/resources/institutional-config.properties [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+logs.directory=/infra/logs
+attachments.directory=/infra/work/attachments
+attachments.pending.directory=/infra/work/attachments/pending
+work.directory=/infra/work
+reports.directory=/infra/work/reports
+staging.directory=/infra/work/staging
+temp.directory=/infra/work/temp
+batch.file.lookup.root.directories=/infra/work/staging;/infra/work/reports
+templates.directory=/infra/work/templates
+attachment.dir.location=/infra/work/attachments
```
Comment (bhutchinson) [inline src/main/resources/institutional-config.properties]: probably, see below comment

**Comment by bhutchinson** [comment]
This is ready for review again.

**Comment by cah292** [comment]
retest this please

**Pull request merged**